src/main/java/Lottery.java

```diff
@@ -0,0 +1,9 @@
+import java.util.Random;
+public class Lottery {
+    private static final Random RANDOM = new Random();
+    public static Ball getRandomBall() {
+        Colors color = ColorsSupplier.getRandomColor();

```

Comment:
Consider renaming `ColorsSupplier` to `ColorSupplier` for consistency and clarity. 🎨